core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorManagerService.js

```diff
@@ -119,24 +119,26 @@ oppia.factory('SimpleEditorManagerService', [
         data.questionList.getBindableQuestion(
           stateName).setBridgeHtml(newHtml);
       },
-      addNewQuestion: function() {
-        // This effectively adds a new multiple-choice interaction to the
-        // latest state in the chain.
-        var lastStateName = (
-          data.questionList.isEmpty() ?
-          SimpleEditorShimService.getInitStateName() :
-          data.questionList.getLastQuestion().getDestinationStateName());
+      addNewQuestion: function(type) {
+        if(type == 'multiple'){
+          // This effectively adds a new multiple-choice interaction to the
+          // latest state in the chain.
```

Comment:
I'm not sure if we want to do this, because we might end up having a lot of cards with multiple choice interactions. It might be better if we bring up a selection of some sort for the creator to choose an interaction. However, we can do this as a stopgap measure for now.